js/src/api/subscriptions/personal.js

```diff
@@ -58,19 +58,7 @@ export default class Personal {
     this._api.personal
       .accountsInfo()
       .then((info) => {
-        const infoKeys = Object.keys(info);
-        const lastKeys = Object.keys(this._lastInfo);
-        let different = false;
-
-        if (infoKeys.length !== lastKeys.length) {
-          different = true;
-        } else {
-          different = !!infoKeys.find((key) => {
-            return (!lastKeys[key] || this._lastInfo[key].name !== info[key].name);
-          });
-        }
-
-        if (different) {
+        if (JSON.stringify(this._lastInfo) !== JSON.stringify(info)) {
```

Comment:
I think comparing to `JSON.stringify()`s is pretty dangerous. If the order of properties is different, this will fail. Also `JSON.stringify` is usually slower than `deepStrictEqual` implementations and it breaks with circular structures. `JSON.stringify` might also fail on objects with a prototype.

[`deep-equal`](https://github.com/substack/node-deep-equal) is very lightweight.